config/crd/bases/networking.x-k8s.io_tlsroutes.yaml

```diff
@@ -32,6 +32,40 @@ spec:
           spec:
             description: TLSRouteSpec defines the desired state of TLSRoute
             properties:
+              gateways:
+                default:
+                  allow: SameNamespace
+                description: Gateways defines which Gateways can use this Route.
+                properties:
+                  allow:
+                    default: SameNamespace
+                    description: 'Allow indicates which Gateways will be allowed to use this route. Possible values are: * All: Gateways in any namespace can use this route. * FromList: Only Gateways specified in GatewayRefs may use this route. * SameNamespace: Only Gateways in the same namespace may use this route.'
+                    enum:
+                    - All
+                    - FromList
+                    - SameNamespace
+                    type: string
+                  gatewayRefs:
+                    description: GatewayRefs must be specified when Allow is set to "FromList". In that case, only Gateways referenced in this list will be allowed to use this route. This field is ignored for other values of "Allow".
+                    items:
+                      description: LocalObjectReference identifies an API object within a known namespace.
+                      properties:
+                        group:
+                          description: Group is the API group name of the referent.
+                          type: string
+                        name:
+                          description: Name is the name of the referent.
+                          type: string
+                        resource:
+                          description: Resource is the API resource name of the referent.
+                          type: string
+                      required:
+                      - group
+                      - name
+                      - resource
```

Comment:
1. How is a user suppose to target a Gateway in another Namespace?
2. Do we really need a Group and Resource here?

Reply:
🤦 thanks for catching this! Should be fixed now.